build.gradle.kts

```diff
@@ -78,40 +114,42 @@ extra.apply {
   ))
   set("sharedParameters", parameters)
   set("linuxParameters", linuxParameters)
-  set("jPackageCmd", jPackageCmd)
+
+  // macOS related stuff
   val uppercaseProjectName = project.name.substring(0,1).toUpperCase() + project.name.substring(1)
   set("uppercaseProjectName", uppercaseProjectName)
   set("appDirName", "${buildDir}/dist/${uppercaseProjectName}.app")
-  set("dmgFilename", "${buildDir}/dist/${project.name}-${project.version}.dmg")
-  set("rpmFilename", "${buildDir}/dist/${project.name}-${project.version}-1.x86_64.rpm")
-  set("debFilename", "${buildDir}/dist/${project.name}_${project.version}-1_amd64.deb")
-  set("msiFilename", "${buildDir}/dist/${project.name}-${project.version}.msi")
 }
 
 tasks.register("createDistDir") {
   group = "build"
-  description = "Creates the directory for distribution"
+  description = "Creates the directory for distribution."
   dependsOn("shadowJar")
   inputs.dir("${buildDir}/libs")
   outputs.dir("${buildDir}/dist")
   doLast {
     if (File("${buildDir}/libs").list().count() != 1) {
-      throw GradleException("${buildDir}/libs should just contain a single shadowJar file.")
+      throw GradleException("${buildDir}/libs should just contain a single shadowJar file. Try \"./gradlew clean\" first.")
     }
     val folder = File("${buildDir}/dist")
     if (!folder.exists() && !folder.mkdirs()) {
-      throw GradleException("Unable to create directory \"${buildDir}/dist\"")
+      throw GradleException("Unable to create directory \"${buildDir}/dist\".")
     }
   }
 }
 
+/*
+ * Task: createDeb
+ *
+ * Creates the Linux DEB package file (Debian, Ubuntu and derrivatives).
+ */
 tasks.register("createDeb") {
   group = "build"
-  description = "Makes the Linux platform specific packages"
+  description = "Makes DEB Linux package file."
```

Comment:
`DEB package` is a kind of file, so the "file" at the end of the line is redundant.